go/consensus/genesis/genesis.go

```diff
@@ -42,6 +43,10 @@ type Parameters struct { // nolint: maligned
 
 	// PublicKeyBlacklist is the network-wide public key blacklist.
 	PublicKeyBlacklist []signature.PublicKey `json:"public_key_blacklist,omitempty"`
+
+	// SoftwareVersion represents the latest consensus-breaking software version
+	// that follows calendar versioning (yy.minor[.micro]).
+	SoftwareVersion *version.Version `json:"software_version,omitempty"`
```

Comment:
As mentioned in the other PR, can we rename this to something like `FeatureVersion`?

Reply:
Missed the comment. Yes, will change.